Kitodo/src/main/java/de/sub/goobi/metadaten/MetadatenImagesHelper.java

```diff
@@ -566,16 +566,17 @@ public int compare(URI firstUri, URI secondUri) {
     }
 
     /**
-     * Get data files.
+     * Get data files. First read them all and next if their size is bigger than
+     * zero sort them with use of GoobiImageFileComparator.
      *
-     * @param myProcess
+     * @param process
      *            Process object
      * @return list of Strings
      */
-    public List<URI> getDataFiles(Process myProcess) throws InvalidImagesException {
+    public List<URI> getDataFiles(Process process) throws InvalidImagesException {
```

Comment:
Can you please adjust the JavaDoc for return value as a list of URI is returned and not a list of String?